mlir/utils/jenkins/Jenkinsfile.downstream

```diff
@@ -7,6 +7,34 @@ void buildProject(String target, String cmakeOpts) {
         cmakeArgs: "-DMLIR_MIOPEN_DRIVER_ENABLED=1 $cmakeOpts"
 }
 
+void buildMIOpen(String cmakeOpts) {
```

Comment:
Not an issue in this PR but we may start to consider to let two jenkinsfile reuse those common utilities in a third file.